frontend/packages/dev-console/integration-tests/support/pageObjects/quickStarts-po.ts

```diff
@@ -0,0 +1,20 @@
+export const quickStartsPO = {
+  quickStartTitle: '[data-test="QuickStartCatalogPageTitle"]',
+  filterKeyword: 'input.pf-c-search-input__text-input',
+  statusFilter: 'button.pf-c-select__toggle',
+  statusDropdown: '[aria-label="Select filter"]',
+  statusComplete: '[data-key="Complete"]',
+  installOpenshiftPipelineOperatorCard:
+    '[data-test="tile-Install the OpenShift Pipelines Operator"]',
+  getStartedWithASampleApplicationCard: '[data-test="tile-Get started with a sample application"]',
+  emptyState: 'div.pf-c-empty-state__content',
+  clearFilter: '[data-test="clear-filter-button"]',
+  cardStatus: '[data-test="quick-start-card-status"]',
```

Comment:
```suggestion
  clearFilter: '[data-test="clear-filters button"]',
  cardStatus: '[data-test~="tile"] [data-test~="status"]',
```

But notice, the card status will return potential all status fields for the complete page.

Reply:
The `cardStatus` will be used with a particular quick start card to show it's status